tests/queries/0_stateless/02353_explain_ast_after_rewrite.sql

```diff
@@ -0,0 +1,4 @@
+-- { echoOn }
+EXPLAIN AST after_rewrite=0 SELECT * FROM numbers(0);
```

Comment:
Maybe it is better to call it `optimize` (like for `PLAN`) or just `rewrite`, what do you think?

Reply:
Yes, I like `rewrite` - it's simpler. Regarding `optimize`, - our current AST manipulation involves optimizations, but I hope overtime it'll be turned more into rewrite than optimization.